src/test/feature/UDF/TestUDF.cpp

```diff
@@ -200,6 +218,10 @@ TEST_F(TestUDF, TestUDFPljava)
 	// run test if pljava language is enabled
 	if (util.getQueryResult("SELECT lanname FROM pg_language WHERE lanname = 'java'") == "java")
 	{
+		// prepare jar files
+		hawq::test::Command cmd("cd " + d_feature_test_root + "/UDF/sql/; javac PLJavaAdd.java; jar cf PLJavaAdd.jar PLJavaAdd.class");
+		cmd.run();
+
```

Comment:
I'm wondering we should add a chapter in README.md for requirement, e.g. we need java dev package for test running here. Others look good.